sig-policy/policy-charter.md

```diff
@@ -0,0 +1,42 @@
+# SIG Policy charter
+
+## Scope
+
+The Policy SIG covers governance capability, security remediation, and configuration management for the Open Cluster
+Management (OCM) project.
+
+In particular, it currently includes the OCM Policy Framework, which is responsible for top-level `Policy` objects and
+syncing those objects between the Hub and managed clusters. It additionally includes the controllers that process the
+down-level policies such as `ConfigurationPolicy`.
+
+### In scope
+
+#### Code, Binaries and Services
+
+- Review and approve policy related OCM enhancements.
+- The APIs in the `policy.open-cluster-management.io` API group. API versioning and deprecation strategy is handled
```

Comment:
I think API versioning and deprecation strategy should be consistent all over the org following the rule defined here https://github.com/open-cluster-management-io/enhancements/tree/main/guidelines#graduation-criteria. 
The deprecation period should follow the same rule.

The sig could decide to deprecate an API, but this should be call out in the community and give the community several weeks in case of any objection. 

any other thought @yue9944882

Reply:
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api_changes.md#alpha-beta-and-stable-versions

@qiujian16 yes in kubernetes we are valuing the API design/compatbility in the first place, b/c kubernetes is data-centric, those PRs involving API-change will be red-marked for more attention. but the alpha version API is sort of exceptional, we dont promise any compatbility/usability b/c they can be rework'ed at any time.